project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -471,13 +486,41 @@ function c4m_user_og_c4m_cta_block() {
 
   $membership_request_value = c4m_og_get_group_membership_request($group);
 
-  $items = array();
-  $items['visitors'] = array(
+  return array(
     '#theme' => 'c4m_group_visitor_cta',
     '#button_label' => $types[$membership_request_value]['button_label'],
     '#button_icon' => $types[$membership_request_value]['button_icon'],
     '#url' => url('/user'),
   );
+}
 
-  return $items;
+/**
+ * Sets the join group link CTA for logged in users that are not members of the
```

Comment:
@DavidHernandez 
This will fail on the sniffer.

first row of the document must be only one row.